GoogleForCreators/web-stories-wp
# Move Selection + change z-index to display selection on top of other layers.

## Summary
Changes z-index to display selection on top of other layers.


## Relevant Technical Choices



## To-do

- [x] Display selection under the Page carousel



## User-facing changes
The selection is now displayed on top of page actions.


## Testing Instructions
1. Add any element
2. Drag it to the bottom of the page.
3. Resize it so that part of the element goes on top of the Page actions but under the Page carousel
4. Verify the selection is still fully visible on top of the Page Actions and the element can be resized from the handles outside of the Page / on top of the Page Actions.


---



Fixes #2463
